lib/FullScreenContainer.js

```diff
@@ -118,14 +115,10 @@ export default class FullScreenContainer extends React.Component {
       return;
     }
 
-    if (Platform.OS === 'ios') {
-      this.flatListView.scrollToIndex({
-        index: index,
-        animated: animated,
-      });
-    } else {
-      this.flatListView.setPageWithoutAnimation(index);
-    }
+  this.flatListView.scrollToIndex({
```

Comment:
Let's start using shorthand version of this assignment:

```
this.flatListView.scrollToIndex({
  index,
  animated,
})
```

Reply:
I'm not sure what you mean by this